guides/source/upgrading_ruby_on_rails.md

```diff
@@ -530,6 +530,16 @@ To:
 Rails.application.secrets[:smtp_settings][:address]
 ```
 
+### Removed deprecated support to `:text` and `:nothing` in `render`.
```

Comment:
Nitpick: None of the other headers are in sentence form. Can you remove the period here?

Reply:
Done :)